shared/team-building/user-result.desktop.tsx

```diff
@@ -1,29 +1,15 @@
 import * as React from 'react'
```

Comment:
I think the desktop version of this component should be ready to render phone numbers / email "results" because users will be able to search for phone number or emails even if they are not in their contacts (or they are not syncing contacts at all).

Reply:
Yeah that's fine, but I still wanted to split it out because the layout is now very different.

Reply:
@zapu does that search work yet?

Reply:
nope it's in my pr and i have some rebasing to do... will get to it tomorrow

Reply:
OK I'll make that a separate PR then.

Reply:
I can also do this change in my PR